.infra/workers.ts

```diff
@@ -249,10 +237,6 @@ export const workers: Worker[] = [
     topic: 'api.v1.source-created',
     subscription: 'api.source-created-squad-user-action',
   },
-  {
-    topic: 'api.v1.source-created',
-    subscription: 'api.source-created-squad-owner-mailing',
-  },
```

Comment:
We don't send this anymore or send it through CIO somehow?

Reply:
CIO has everything needed 😬